### PR TITLE
scraper: support bearer token with sso configured credentials

### PR DESCRIPTION
**Pull request by masha1997**

### CODE REVIEW

File: scraper/main.go [hunk 1]
```diff
@@ -47,8 +52,17 @@ func main() {
 
 	models.Verbose = (*verbose == "true")
 	models.Cache = models.NewCache(1000)
+	bearerToken := ""
+
+	if !(*tokenPath == "" || *profile == "" || *region == "") {
+		bearerToken, err = getTokenFromSSM(*tokenPath, *profile, *region)
```
Comment (benoittoulme) [inline scraper/main.go:58]: maybe we should validate that all of the 3 variables are set.

**Review by dmitrykruglov — APPROVED** [review]
Looks good 👍

File: scraper/main.go (hunk 1 above)
Comment (masha1997) [reply scraper/main.go:63]: added commands to  'in-built' validate combinations of options and auth modes:
```
creds --user admin --password test --format tf --verbose true -s true
token --region us-west-2 --profile tivo-oi --path /tivooi/itsi_validator_tf_access_token --format tf --verbose true -s true 
```

Comment (benoittoulme) [reply scraper/main.go:63]: nice. I did not know about parser.NewCommand, that is very useful.

**Review by benoittoulme — APPROVED** [review]

**Pull request merged**